app/api/mapping/services.py

```diff
@@ -68,3 +70,36 @@ def delete_blob(blob_name: str, container: str) -> bool:
     # Delete the blob
     blob_dict_client.delete_blob()
     return True
+
+
+def modify_filename(filename: str, dt: str, rand: str) -> str:
+    split_filename = os.path.splitext(str(filename))
+    return f"{split_filename[0]}_{dt}_{rand}{split_filename[1]}"
+
+
+def upload_files(sr_id, sr_name, sr_file, dict_name, dict_file: Optional[any] = None):
```

Comment:
```suggestion
def upload_blob(sr_id, sr_name, sr_file, dict_name, dict_file: Optional[any] = None):
```
Also we'll need type annotations, and a doc string for this method.

It also needs to be more abstract, so it's a reusable method for any file upload to Azure, not just these specific files.